src/main/java/org/jenkinsci/maven/plugins/hpi/TestInsertionMojo.java

```diff
@@ -47,6 +47,11 @@ private static String quote(String s) {
     }
 
     public void execute() throws MojoExecutionException, MojoFailureException {
+        if (!project.getPackaging().equals("hpi")) {
```

Comment:
NIT: I prefer always put the static string on the left side to make NPE impossible
`if ( !"hpi".equals(project.getPackaging()) ) {`